pontoon/contributors/templates/contributors/settings.html

```diff
@@ -64,6 +64,18 @@ <h3>Personal information</h3>
         </div>
       </section>
 
+   <section class="appearance" >
+        <h3>Appearance</h3>
+        <div class="field">
+            <p class="help">Choose if appearance should be dark, light, or follow your system’s settings</a></p>
+        </div>
+        <span class="toggle-button">
+            <button type="button" class="dark active" title="Use a dark theme"><i class="icon far fa-moon"></i>Dark theme</button>
```

Comment:
We need to dynamically set `active` instead of hard-coding it.

Inspiration:
https://github.com/mozilla/pontoon/blob/main/pontoon/base/templates/widgets/toggle.html